lib/views/pages/new_post/new_post_form.dart

```diff
@@ -92,7 +78,12 @@ class NewPostForm extends HookConsumerWidget {
               fit: FlexFit.loose,
               child: _verticallyPadded(bodyField),
             ),
-            _verticallyPadded(buttonRow),
+            _verticallyPadded(
+              SizedBox(
+                width: double.infinity,
+                child: postButton,
+              ),
```

Comment:
This used to be an `Expanded`. I'm guessing you decided to use this `SizedBox` for the widget to take the maximum width. 
However, I have to admit I took a bit of time to understand exactly what `width: double.infinity` meant in this scenario. I tried the following that works exactly the same:
```suggestion
              FractionallySizedBox(
                widthFactor: 1.0,
                child: postButton,
              ),
```

What do you think? Does it make the code more readable?

Reply:
For context, I switched from `Expanded` to `SizedBox` because `Expanded` has to be in a parent widget such as `Row`, which didn't make any sense after we had removed the settings icon.

For `SizedBox` vs `FractionallySizedBox`, I think the opposite of you haha, I feel like `SizedBox` is more straightforward and `FractionallySizedBox` should only be used if we want to set a fraction that is not 1. I think in both cases, it might take a little bit of time for the reader to understand.

Let me know what you think, and we can always ask for a third opinion :)

Reply:
As we discussed on our group chat, it appears @gruvw discussed with a developer on this subject one should use a `Row` with a single `Expanded` child here.

Reply:
Yup, fixed in 44b48a8f55db6932fc8aa9ac5d4e478c27a864a4.